cmd/operator/kodata/knative-serving/0.15.1/1-serving-crds.yaml

```diff
@@ -0,0 +1,602 @@
+# Copyright 2019 The Knative Authors
```

Comment:
Should we just always apply the CRDs early in the Operator logic?

Reply:
We can do that in the reconciler, but reconciler needs to look up the CRD by the label `knative.dev/crd-install: "true"`.